plugins/toolbox/config.d.ts

```diff
@@ -0,0 +1,55 @@
+import {CornerDotType, CornerSquareType, DotType, ShapeType} from "qr-code-styling";
```

Comment:
Hm. Not sure it's ok to import these in the schema. Also could change to `import type` instead `import`

Reply:
I don't think it works as I hoped it would. The `backstage-cli config:check` does not report a value that does not match the type 😢 

The guidelines about config say to limit the use of configuration as much as possible. In this case it is not per environment but to pass config to the plugin. 

The other option I see is to pass the toolbox config as a prop to the ToolboxPage or something, but I don't think it will cover the homepage use case then?